### PR TITLE
Add controller level commands tests.

### PR DESCRIPTION
**Pull request by anastasiamac**
Straight forward port of https://github.com/juju/juju/pull/9797.

### CODE REVIEW

**Comment by anastasiamac** [comment]
self-approving as a straight forward-port.

**Comment by anastasiamac** [comment]
$$merge$$

**Comment by anastasiamac** [comment]
[current controller saga](https://trello.com/c/yWRMkBM6/620-current-controller-saga)

File: featuretests/cmd_juju_current_controller_test.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	"github.com/juju/loggo"
+	jc "github.com/juju/testing/checkers"
+
+	"github.com/juju/juju/cmd/juju/commands"
+	jujutesting "github.com/juju/juju/juju/testing"
+	"github.com/juju/juju/jujuclient"
+)
+
+type cmdCurrentControllerSuite struct {
+	jujutesting.JujuConnSuite
```
Comment (SimonRichardson) [inline featuretests/cmd_juju_current_controller_test.go]: Is this correct?

Comment (anastasiamac) [reply featuretests/cmd_juju_current_controller_test.go]: yes. it's in featuretest pkg for a reason :)

**Comment by anastasiamac** [comment]
$$merge$$

**Pull request merged**